scene/main/node.cpp

```diff
@@ -1348,6 +1348,19 @@ Node *Node::get_parent() const {
 	return data.parent;
 }
 
+Node *Node::find_parent(const String &p_mask) const {
```

Comment:
Argument should be StringName, which is just a pointer comparison with p->data.name and hence a lot faster. String is an alphanumerical comparison and considerably slower.

Reply:
Other than that, PR is fine and I can see it being useful.

Reply:
From a short discussion with reduz, you're right and in this case, not using `StringName` is fine.